streamer/src/nonblocking/testing_utilities.rs

```diff
@@ -84,27 +87,20 @@ pub struct SpawnTestServerResult {
 }
 
 pub fn create_quic_server_sockets() -> Vec<UdpSocket> {
-    #[cfg(not(target_os = "windows"))]
-    {
-        use {
-            solana_net_utils::bind_to,
-            std::net::{IpAddr, Ipv4Addr},
-        };
-        (0..10)
-            .map(|_| {
-                bind_to(
-                    IpAddr::V4(Ipv4Addr::LOCALHOST),
-                    /*port*/ 0,
-                    /*reuseport:*/ true,
-                )
-                .unwrap()
-            })
-            .collect::<Vec<_>>()
-    }
-    #[cfg(target_os = "windows")]
-    {
-        vec![bind_to_localhost().unwrap()]
-    }
+    let num = if cfg!(not(target_os = "windows")) {
+        10
+    } else {
+        1
+    };
+    let port_range = localhost_port_range_for_tests();
+    multi_bind_in_range_with_config(
+        IpAddr::V4(Ipv4Addr::LOCALHOST),
+        port_range,
+        SocketConfig::default().reuseport(true),
+        num,
+    )
+    .expect("bind operation for quic server sockets should succeed")
+    .1
```

Comment:
I'm not in favour of using tuples here because `.1` is not very descriptive.

Reply:
Changing the public API of net_utils is not on the menu just yet, another PR.